File: MainActivity.java
package com.example.fuelextimateapp;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);
        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main), (v, insets) -> {
            Insets systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars());
            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom);
            return insets;
        });
        //1-Create object
        Button btnCost;
        EditText txtDist;
        EditText txtEffc;
        EditText txtPrice;
        TextView txtCost;

        //2-Bind UI to Java object
        btnCost = findViewById(R.id.btnCost);
        txtCost = findViewById(R.id.txtFuelCost);

        txtDist = findViewById(R.id.TxtDist);
        txtEffc = findViewById(R.id.txtEffc);
        txtPrice = findViewById(R.id.txtPrice);

        btnCost.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String dist = txtDist.getText().toString();
                String effc = txtEffc.getText().toString();
                String price = txtPrice.getText().toString();

                int d = Integer.getInteger(dist);
                int e = Integer.getInteger(effc);
                int p = Integer.getInteger(price);

                int ans =  (d / e) * p;

                txtCost.setText("" + ans + "PKR");
            }
        });
    }
}

File: MainActivity.kt
package com.example.myandroid

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import org.w3c.dom.Text

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_main)

        var tvText = findViewById<TextView>(R.id.tvText)
        var btnSubmit = findViewById<Button>(R.id.btnSubmit)
        var etText = findViewById<EditText>(R.id.etText)

       btnSubmit.setOnClickListener { var etString = etText.text.toString().trim()

       tvText.text = "Hello $etString!"
       }



    }
}